packages/formation-react/src/components/Table/Table.jsx

```diff
@@ -51,9 +51,15 @@ function Table(props) {
             {fields.map((field, index) => (
               <td
                 data-index={index}
-                className={classNames(borderClasses, {
-                  'vads-u-text-align--left': field.alignLeft,
-                })}
+                className={classNames(
+                  borderClasses,
+                  {
+                    'vads-u-text-align--left': field.alignLeft,
+                  },
+                  {
+                    'medium-screen:vads-u-text-align--right': field.alignRight,
+                  },
+                )}
```

Comment:
These can be combined into the same object which reads easier in my opinion.

An [example based on the docs](https://github.com/JedWatson/classnames#usage):

```js
const borderClasses = 'test';

classNames(borderClasses,
    { 
        foo: true
    }, 
    { 
        bar: true
    }
); // => 'test foo bar'

classNames(borderClasses,
    { 
        foo: true, 
        bar: true 
    }
); // => 'test foo bar'
```

Reply:
Yep, updated this